purchases/src/testDefaults/kotlin/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -524,6 +524,28 @@ internal class PurchasesTest : BasePurchasesTest() {
         }
     }
 
+    @Test
+    fun `login called with different appUserID notifies backup manager`() {
```

Comment:
maybe we need a test for logout as well?

Reply:
I kinda reused the existing `logout called with identified user makes right calls` test, which tests a few things... Might be worth splitting up in multiple tests, but I think that might be better to do on a separate PR?